semeio/jobs/design_kw/design_kw.py

```diff
@@ -73,6 +80,10 @@ def is_perl(file_name, template):
     return file_name.endswith(".pl") or template[0].find("perl") != -1
 
 
+def is_xml(file_name, template):
+    return file_name.endswith(".xml") or template[0].find("xml") != -1
```

Comment:
the `template[0].find("xml")` I guess is copied from `is_perl()` where it is used as a heuristic to try to catch  for example
the shebang in a perl file (that would have made a better example in the test code for perl). Is there a similar probability for the string "xml" in the first line in an xml-file?  Could we risk ruining something where the letters `xml` is found on the first line? (no not really, since this is only changing from error to warning)

Reply:
Usually there is a string of type <?xml version="1.0" ?> in the heading. Should I check on ?xml instead maybe ? It could be useful that the code identify a file as xml file also from the top string and not only from the extension .xml What do you think? Use ?xml as search word instead of xml in the heading?

Reply:
It works with '?xml' as search word also to identify a file as xml format.